.github/workflows/build.yml

```diff
@@ -22,7 +22,12 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'yarn'
 
-      - name: Install dependencies
+      - name: Install dependencies (Node 16.x)
+        if: matrix.node-version == '16.x'
+        run: yarn install --frozen-lockfile --ignore-engines
```

Comment:
`jsdom` doesn't support Node v16 and throws an error when running `yarn install`, but the `--ignore-engines` flag allows us to bypass this error and the package still builds